### ISSUE TITLE
Godot Editor 3.5 Android | Freeze and crash when adding nodes to scene.

### ISSUE THREAD

**Issue by LettucePie** [godotengine/godot#66604]
### Godot version

3.5

### System information

Android 10 Samsung Galaxy Tab S4 (Wi-Fi)

### Issue description

I cannot add child nodes to nodes in a scene. I can create new scenes, and choose their starting Node type. Interestingly, I found a "fix" but it's very disruptive. If I go to the app details and clear the Data and Cache, I can then add One child node. After that it will freeze and eventually prompt me to wait or close the app.

I'm not positive if it's the act of adding a Node causing problems or trying to open the window displaying the Nodes that could be added.

I will concede, this tablet is rather old now, and it has Syncthing running in the background all the time. If my memory or hardware has become worn down I wouldn't be surprised. However, I find it especially strange how I can get most of the Editor to work once, but only once.

### Steps to reproduce

1. Uninstall the 3.5 Godot Engine App
2. Install the 3.5 Godot Engine App
3. Launch the App
4. Create a new Scene of any type.
5. Add a child Node to the Parent Node.
6. Add another child Node, and it will freeze.

For the Temporary Fix --

1. Go to the App Details, and then navigate to Storage.
2. Clear the Data/Cache
3. Re-open the App.
4. Find the project folder.
5. Open the Scene and Add a child Node.
6. Save and Repeat from Step 1.

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/64166 (fixed in [3.5.1](https://godotengine.org/article/maintenance-release-godot-3-5-1)).
